scripts/tools-versions-win10.yml

```diff
@@ -0,0 +1,25 @@
+python:
```

Comment:
note: I see a lot of duplication here, should we have a common file, that is imported by OS specific files where versions can be overriden if needed?

Reply:
It is not possible at this moment.
To do so, changes in toolchain bundler are required. Only platform specific files are evaluated during bundle preparation process. They are also used to calculate bundle checksum, so without changes in bundler we cannot extract common requirements to another file